includes/class-wc-gateway-swedbank-pay-cc.php

```diff
@@ -792,11 +739,20 @@ public function payment_confirm() {
 	public function return_handler() {
 		$raw_body = file_get_contents( 'php://input' );
 
-		$this->log( sprintf( 'Incoming Callback: Initialized %s from %s', $_SERVER['REQUEST_URI'], $_SERVER['REMOTE_ADDR'] ) );
-		$this->log( sprintf( 'Incoming Callback. Post data: %s', var_export( $raw_body, true ) ) );
+		$this->adapter->log(
+			LogLevel::INFO,
+			sprintf( 'Incoming Callback: Initialized %s from %s', $_SERVER['REQUEST_URI'], $_SERVER['REMOTE_ADDR'] )
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_SERVER Superglobal detected.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)